cluster/manifests/aws-node-decommissioner/01-rbac.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: "aws-node-decommissioner"
+  namespace: "default"
+  annotations:
+    eks.amazonaws.com/role-arn: "arn:aws:iam::{{ .Cluster.InfrastructureAccount | getAWSAccountID }}:role/app-aws-node-decommissioner"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
I wonder if a `Role` really works here or if it should be `ClusterRole` considering that we're patching non-namespaced resource `nodes`?

Reply:
True, will modify to ClusterRole